customer-commons/src/main/java/no/unit/nva/customer/model/channelclaim/ChannelClaimWithClaimer.java

```diff
@@ -0,0 +1,7 @@
+package no.unit.nva.customer.model.channelclaim;
+
+import java.net.URI;
+
+public record ChannelClaimWithClaimer(ChannelClaimDto channelClaim, URI customerId, URI cristinId) {
```

Comment:
I guess we have used "xId" elsewhere, this was a mistake, but maybe a consistent one.